float32 elevation_max  #meter
float32 elevation_min  #meter
float32 x_size  #meter
float32 y_size  #meter
float32 origin_x  #meter
float32 origin_y  #meter
float32 cost_resolution #cost/meter
float32 zero_elevation_cost  #cost
